src/2025h1/build-std.md

```diff
@@ -0,0 +1,150 @@
+  of build-std and give the responsible upstream teams confidence in our ownership of this
+  feature, and enabling those teams to make a fully informed decision on any proposals made.
+
+- Next, after and conditional on acceptance of this RFC, we will proceed with its
+  implementation.
+
+The fifty-thousand foot view of the work involved in this feature is:
+
+- Having the standard library sources readily available that match the compiler.
+- Being able to build those sources without using a nightly toolchain, which has many
+  possible solutions.
+- Having a blessed way to build at least `core` without Cargo, which some users like
+  Rust for Linux would like.
+  - This would be optional but may be a side-effect of whatever mechanism for build-std
+    the MVP RFC eventually proposes.
+- Being able to tell the compiler to use the resulting prebuilt standard library sources
+  instead of the built-in standard library, in a standard way.
+- Integrating all of the above into Cargo.
+- Making sure all of this works for targets that don't have a pre-built std.
+
+Some of this work exists in the current perma-unstable `-Zbuild-std` implementation, which
+may be re-used if appropriate.
+
+Prior to the submission of this goal, this goal has been discussed with the cargo team and
+leads of the compiler and library teams, ensuring that this goal's owners have liaisons from
+stakeholder teams and the support of the primary teams involved in the design and
+implementation.
+
+### The "shiny future" we are working towards
+
+After the approval and implementation of the MVP RFC, there will naturally be follow-up use cases
+which can be designed and implemented to complete the build-std feature.
+
+## Design axioms
+
+- Enabling build-std without changing any compilation options or configuration should produce an
+  equivalent library to that distributed by the project.
+- Avoid precluding future extensions to build-std.
+- build-std should allow std/alloc/core to be treated more like other dependencies than currently.
+  - This represents a general move away from treating std/alloc/core as a special case.
+
+## Ownership and team asks
+
+| Subgoal                                        | Owner(s) or team(s)  | Notes |
+| ---------------------------------------------- | -------------------- | ----- |
+| Discussion and moral support                   | ![Team][] [cargo]    |       |
+| Stabilise MVP of build-std                     | @adamgemmell         |       |
+| ↳ Author RFC                                   | @adamgemmell         |       |
+| ↳ Implementation                               | @adamgemmell         |       |
+| ↳ Standard reviews                             | ![Team][] [cargo]    |       |
+| ↳ RFC decision                                 | ![Team][] [cargo]    |       |
+| ↳ Author stabilization report                  | @adamgemmell         |       |
+| ↳ Stabilization decision                       | ![Team][] [cargo]    |       |
```

Comment:
```suggestion
| Subgoal                                        | Owner(s) or team(s)  | Notes |
| ---------------------------------------------- | -------------------- | ----- |
| Discussion and moral support                   | ![Team][] [cargo]    |       |
| Stabilise MVP of build-std                     | @adamgemmell         |       |
| ↳ Author RFC                                   | @adamgemmell         |       |
| ↳ Implementation                               | @adamgemmell         |       |
| ↳ Standard reviews                             | ![Team][] [cargo]    |       |
```

I'm weakening these goals based on what you wrote above. My assumption is that your plan is to have the RFC *opened* (not decided upon) and implementation to have begun. Therefore you are not expecting the cargo team to reach an RFC decision or a stabilization decision.

Reply:
Fixed